src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderStateController.php

```diff
@@ -70,6 +70,14 @@ public function indexAction(
             $this->buildFiltersParamsByRequest($request, $orderReturnStatesFilters)
         );
 
+        if ($this->get('prestashop.adapter.multistore_feature')->isActive()) {
+            $this->addFlash(
+                'info',
+                $this->trans('Note that this page is available in all shops context only, this is why your context has just switched.',
+                    'Admin.Notifications.Info')
+            );
```

Comment:
```suggestion
                $this->trans(
                    'Note that this page is available in all shops context only, this is why your context has just switched.',
                    'Admin.Notifications.Info'
                )
            );
```
Better indentation, thinking something was missing :sweat_smile:

Reply:
@matks Suggestion not implemented